### ISSUE TITLE
Add terrafirmacraft ores to autominer support list

### ISSUE THREAD

**Issue by netherever** [age-series/ElectricalAge#195]
Scanner (portable and autominer) don't see TFC ores - therefore, no automaining/
This is sad.

**Comment by jrddunbr** [comment]
Hmmm. I'm pretty sure those were reported working a while back with the autominer. I have screenshots of a world with a working TFC autominer, at least.

**Comment by jrddunbr** [comment]
I'll look into it.

**Comment by netherever** [comment]
It's pretty obviously - ores from tfc very different to vanilla, so scanner see just rock.

**Comment by jrddunbr** [comment]
No, I'm aware of that. I think there's a config or something like that because it worked previously and I haven't made any recent changes to the logic it uses.

**Comment by netherever** [comment]
No, already checking source - it's just can't work with tfc ores.
Obviously - this never tried.

**Comment by jrddunbr** [comment]
I swear, I've seen it done before. But if you're familiar with the source at this point, feel free to check out 1.15.x stable and submit a pull request for it. I'm sure it's not many lines.

**Comment by netherever** [comment]
>I'm sure it's not many lines.
Wrong.
At least need to use tfc api.
But let's take it ease.
Just take as ore any block with entity - this be enough.

**Comment by jrddunbr** [comment]
> feel free to check out 1.15.x stable and submit a pull request for it.

If you don't submit something, I'll look at it probably this weekend, but I can't guarantee that I'll get it working because I'm pretty busy these days. :/

**Comment by netherever** [comment]
Done (hope will work)

**Comment by jrddunbr** [comment]
I'll test things out tonight. I'm concerned about one part of the code, I'm not convinced that it won't just take any tile entity and just mark it as an ore. That would be bad. I'll verify tonight.

**Comment by netherever** [comment]
It's not right thing, true.
But not so bad - work only if set in config, and see (and mined) chest by xray is sort of fun.

**Comment by jrddunbr** [comment]
This issue is being closed now since there will be no more feature updates in Electrical Age for Minecraft 1.7.10.
If the issue is relevant to 1.12, it will be re-opened there at a later date.
